corehq/apps/export/templates/export/partials/export_list_controller.html

```diff
@@ -47,6 +47,7 @@
                                     slug: 'export-list',
                                     perPage: itemsPerPage,
                                     totalItems: totalItems,
+                                    watchChanges: showPagination,
```

Comment:
What's happening here?  Is `showPagination` `false` in some circumstances?

Reply:
Yes. [export_list.js](https://github.com/dimagi/commcare-hq/blob/53c5c0cb9308069fb50f26750982fbc646e6750f/corehq/apps/export/static/export/js/export_list.js#L235) sets it to `!self.isLoadingPanel() && !self.hasError() && self.exports().length`, so it becomes true when the list has been populated.

Reply:
I'm not delighted with the name "watchChanges". "isReady" might give more of a clue what it's going to do.